examples/sample-input-byo.tfvars

```diff
@@ -0,0 +1,118 @@
+# !NOTE! - These are only a subset of variables.tf provided for sample.
+# Customize this file to add any variables from 'variables.tf' that you want 
+# to change their default values. 
+
+# ****************  REQUIRED VARIABLES  ****************
+# These required variables' values MUST be provided by the User
+prefix                                  = "<prefix-value>"
+location                                = "<aws-location-value>" # e.g., "us-east-1"
+ssh_public_key                          = "~/.ssh/id_rsa.pub"
+# ****************  REQUIRED VARIABLES  ****************
+
+# Bring your own existing resources
+vpc_id  = "<existing-vpc-id>" # only needed if using pre-existing VPC
+subnet_ids = {  # only needed if using pre-existing subnets
+  "public" : ["existing-public-subnet-id1", "existing-public-subnet-id2"],
+  "private" : ["existing-private-subnet-id1", "existing-private-subnet-id2"],
+  "database" : ["existing-database-subnet-id1", "existing-database-subnet-id2"]] # only when 'create_postgres=true' 
```

Comment:
There is an extra bracket ] at the end of this line.

Reply:
Fixed!